src/main/default/lwc/flowProgressIndicator/flowProgressIndicator.html

```diff
@@ -0,0 +1,16 @@
+<template>
+  <lightning-progress-indicator
+    current-step={_currentStep}
```

Comment:
Should you be using the internal value `_currentStep`? Or the value returned by the `get currentStep()` you made?